mage_ai/io/oracledb.py

```diff
@@ -8,6 +9,8 @@
 from mage_ai.io.config import BaseConfigLoader, ConfigKey
 from mage_ai.io.sql import BaseSQL
 
+LOGGER = logging.getLogger(__name__)
```

Comment:
Please use logger from this https://github.com/mage-ai/mage-ai/blob/master/mage_ai/server/logger.py#L10
Example: https://github.com/mage-ai/mage-ai/blob/master/mage_ai/server/server.py#L122

Reply:
Updated to use `mage_ai/server/logger.py` logger